### PR TITLE
Rename to "Flutter DevTools"

### PR DESCRIPTION
**Pull request by helin24**
This change addresses https://github.com/flutter/devtools/issues/2345

Screen Shot 2021-01-13 at 9 55 41 AM
Screen Shot 2021-01-13 at 10 01 35 AM

### CODE REVIEW

**Comment by stevemessick** [comment]
I think the UI affordance only shows up after the Flutter app is launched. So it won't be "Dart". But the hover text for the Dart icon should have the same name IMO.

You might want to change the description and text in plugin.xml, just for consistency. It is defined twice, and I'm not sure why.

File: src/io/flutter/run/OpenDevToolsAction.java [hunk 1]
```diff
@@ -34,7 +35,11 @@ public OpenDevToolsAction() {
   }
 
   public OpenDevToolsAction(@NotNull final FlutterApp app, @NotNull final Computable<Boolean> isApplicable) {
-    super("Open DevTools", "Open Dart DevTools", FlutterIcons.Dart_16);
+    super(
+      "Open DevTools",
```
Comment (kenzieschmoll) [inline src/io/flutter/run/OpenDevToolsAction.java:39]: can we make this text the same?
` "Open " + (FlutterModuleUtils.declaresFlutter(app.getProject()) ? "Flutter" : "Dart") + " DevTools",`

Comment (jacob314) [reply src/io/flutter/run/OpenDevToolsAction.java:39]: This is the Flutter plugin so I don't think you can actually run an app unless it declares Flutter.

**Comment by helin24** [comment]
> It is defined twice, and I'm not sure why.

I tried removing it from the `OpenDevToolsAction` but then there is no icon, title, or description. Not sure why..

**Review by kenzieschmoll — APPROVED** [review]

**Pull request merged**